newIDE/app/src/InAppTutorial/Tutorials/OnboardingTutorial.js

```diff
@@ -0,0 +1,482 @@
+// @flow
+
+import { t } from '@lingui/macro';
+import { type InAppTutorial } from '../InAppTutorialContext';
+
+const inAppTutorial: InAppTutorial = {
+  id: 'onboarding',
+  editorSwitches: {
+    GoToBuildSection: 'Home',
+    ClickOnNewObjectButton1: 'Scene',
+    ClickOnNewEvent: 'EventsSheet',
+  },
+  endDialog: {
+    content: [
+      {
+        messageDescriptor: t`## Congratulations! 🎉`,
+      },
+      {
+        messageDescriptor: t`### You’ve built your first game! 😊`,
+      },
+      {
+        messageDescriptor: t`You’re now ready to learn the basics of GDevelop.`,
+      },
+      {
+        messageDescriptor: t`Click the image to start!`,
+      },
+      {
+        messageDescriptor: t`👇👇👇`,
+      },
+      {
+        cta: {
+          imageSource: 'https://i3.ytimg.com/vi/bR2BjT7JG0k/mqdefault.jpg',
+          linkHref:
+            'https://www.youtube.com/watch?v=bR2BjT7JG0k&list=PL3YlZTdKiS89Kj7IQVPoNElJCWrjZaCC8',
+        },
+      },
+      {
+        messageDescriptor: t`### Want to skip the basics?`,
+      },
+      {
+        messageDescriptor: t`Go to the "Learn" section on the app to explore advanced materials.`,
+      },
+      {
+        messageDescriptor: t`Have fun!`,
+      },
+    ],
+  },
+  flow: [
+    {
+      id: 'GoToBuildSection',
+      elementToHighlightId: '#home-build-tab',
+      nextStepTrigger: { presenceOfElement: '#home-create-project-button' },
+      tooltip: {
+        description: {
+          messageDescriptor: t`Head over to the **Build section**`,
+        },
+        placement: 'right',
+      },
+    },
+    {
+      id: 'CreateProject',
+      elementToHighlightId: '#home-create-project-button',
+      nextStepTrigger: { presenceOfElement: '#create-project-button' },
+      tooltip: {
+        description: {
+          messageDescriptor: t`We'll create a simple game with **a character that can collect coins**.
+          \n\nLet's create a new project!`,
+        },
+      },
+    },
+    {
+      id: 'ValidateProjectCreation',
+      elementToHighlightId: '#create-project-button',
+      nextStepTrigger: {
+        presenceOfElement: '#tab-layout-Untitled-scene-button',
+      },
+      tooltip: {
+        description: { messageDescriptor: t`Let's go!` },
+      },
+      isOnClosableDialog: true,
+    },
+    {
+      id: 'ClickOnNewObjectButton1',
```

Comment:
should we avoid numbers in IDs? otherwise we'll end up with `ClickOnNewObjectButton25`

Maybe `ClickOnNewObjectButtonForCharacter` & `ClickOnNewObjectButtonForCoin`?

Reply:
I'm not sure it's really important. This is not really part of the code, it's more data that we won't control the day someone out of the team creates a tutorial.
But I'll change it here!